sections/video.liquid

```diff
@@ -0,0 +1,128 @@
+{{ 'video-section.css' | asset_url | stylesheet_tag }}
+{{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+
+<div class="video-section{% unless section.settings.full_width %} page-width{% endunless %}">
+  <h2 class="title{% if section.settings.full_width %} page-width{% endif %}">{{ section.settings.heading }}</h2>
+  <div{% if section.settings.image_padding %} class="video-section-padding"{% endif %}>
+    <noscript>
```

Comment:
I noticed in the demo store there's an extra, empty `a` being generated somehow:

```html
<noscript>
  <div class="video-section__media" style="padding-bottom: 66.67480468750001%">
    <a
      href="https://vimeo.com/571010526"
      class="video-section__poster media media--transparent"
      style="padding-bottom: 66.67480468750001%"
    >
      <img
        srcset="…"
        src="//cdn.shopify.com/s/files/1/0458/4099/0230/files/a-persons-hands-knitting-with-white-wool_533x.jpg?v=1625665055"
        sizes="(min-width: 990px)990px, (min-width: 750px) calc(100vw - 100px), calc(100vw - 30px)"
        alt="Describe the video for customers using screen readers"
        loading="lazy"
        width="4096"
        height="2731"
        class="video-section__image"
      />
    </a>
  </div>
  <a
    href="https://vimeo.com/571010526"
    class="video-section__poster media media--transparent"
    style="padding-bottom: 66.67480468750001%"
  ></a>
</noscript>
```